dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/bnd.bnd

```diff
@@ -28,7 +28,9 @@ fat.project: true
 
 fat.test.container.images: \
     jaegertracing/all-in-one:1.39, \
-    openzipkin/zipkin-slim:2.23
+    openzipkin/zipkin-slim:2.23, \
+    otel/opentelemetry-collector:0.74.0 ,\
+    alpine:3.16
```

Comment:
I don't see where we're using apline:3.16

Reply:
I get this error when I do not include this: 

```
Used testcontainer image(s) [alpine:3.16] were not found in the fat.test.container.images property! To correct this, add fat.test.container.images: alpine:3.16 to the bnd.bnd file for this FAT so that a testcontainer image graph can be generated in the future.

java.lang.IllegalStateException: Used testcontainer image(s) [alpine:3.16] were not found in the fat.test.container.images property! To correct this, add fat.test.container.images: alpine:3.16 to the bnd.bnd file for this FAT so that a testcontainer image graph can be generated in the future.
at componenttest.containers.ImageVerifier.assertImages(ImageVerifier.java:85)
at componenttest.containers.TestContainerSuite$1.after(TestContainerSuite.java:59)
```

I know that this is used as the base image for the opentelemtry collector: 
https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/Dockerfile

Reply:
Huh, weird. Thanks for checking.